### PR TITLE
extra-container: new section and packages

### PR DESCRIPTION
**Pull request by szclsya**

### CODE REVIEW

File: extra-container/buildah/autobuild/build [hunk 1]
```diff
@@ -0,0 +1,8 @@
+abinfo "Fetching source..."
+git clone https://github.com/containers/buildah
```
Comment (MingcongBai) [inline extra-container/buildah/autobuild/build]: Why not do this in `spec`?

File: extra-container/buildah/autobuild/build [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+cd "$SRCDIR"/buildah
+git checkout "tags/v$PKGVER"
+
+abinfo "Building buildah..."
+make PREFIX=/usr
+make DESTDIR="$PKGDIR" PREFIX=usr install install.completions
```
Comment (MingcongBai) [inline extra-container/buildah/autobuild/build]: You could have easily used the `plainmake` ABTYPE here. Leave `install.completions` in `beyond`.

File: extra-container/buildah/spec [hunk 3]
```diff
@@ -0,0 +1,2 @@
+VER=1.15.1
+DUMMYSRC=1
```
Comment (MingcongBai) [inline extra-container/buildah/spec]: Leave your Git source specification here.

File: extra-container/cni-plugins/autobuild/build [hunk 4]
```diff
@@ -0,0 +1,10 @@
+cd "$SRCDIR"
+
+abinfo "Building cni-plugins..."
+./build_linux.sh
```
Comment (MingcongBai) [inline extra-container/cni-plugins/autobuild/build]: Use absolute path.

File: extra-container/cni-plugins/autobuild/build [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+
+abinfo "Installing cni-plugins binaries..."
+cd "$SRCDIR"/bin
+for bin in *; do
+  install -Dm755 $bin "$PKGDIR/usr/lib/cni/$bin"
```
Comment (MingcongBai) [inline extra-container/cni-plugins/autobuild/build]: Verbose this with `install -Dvm755`.

File: extra-container/cni-plugins/autobuild/defines [hunk 6]
```diff
@@ -0,0 +1,4 @@
+PKGNAME=cni-plugins
+PKGDES="Some CNI network plugins, maintained by the containernetworking team."
+PKGSEC=container
+
```
Comment (MingcongBai) [inline extra-container/cni-plugins/autobuild/defines]: Missing dependency declaration(s)?

**Review by MingcongBai — CHANGES_REQUESTED** [review]

File: extra-container/buildah/autobuild/build (hunk 1 above)
Comment (szclsya) [reply extra-container/buildah/autobuild/build]: buildah will use git history during build, so a full repository is needed,

Comment (MingcongBai) [reply extra-container/buildah/autobuild/build]: > buildah will use git history during build, so a full repository is needed,

In this case, still use GITSRC + GITCO, then in prepare, invoke `acbs_copy_git`.

**Comment by MingcongBai** [comment]
@szclsya ping?

**Pull request closed without merging**